actions/subscribers.go

```diff
@@ -105,6 +106,25 @@ func PostSubscriber(c *gin.Context) {
 		return
 	}
 
+	boundariesvc := boundaries.New(storage.GetFromContext(c))
+
+	limitexceeded, _, err := boundariesvc.SubscribersLimitExceeded(middleware.GetUser(c))
```

Comment:
Why do you send the user you just need the id

Reply:
i also need user.boundaries.SubLimit

Reply:
oh i didnt saw that. I am resolving this